src/Explorer/Tables/DataTable/DataTableBindingManager.ts

```diff
@@ -1,15 +1,14 @@
 import * as ko from "knockout";
 import * as _ from "underscore";
 
+import QueryTablesTab from "../../Tabs/QueryTablesTab";
```

Comment:
Let's keep an eye on this kind of reordering of imports. Unless you are getting a lint error, undo these kinds of changes as they produce code churn for no benefit. No need to fix it here. Just keep an eye on it in future PRs.

Reply:
I will keep an eye on these changes for future changes